packages/aws-cdk-lib/aws-efs/README.md

```diff
@@ -96,14 +96,38 @@ fileSystem.addToResourcePolicy(statement);
 ### Permissions
 
 If you need to grant file system permissions to another resource, you can use the `.grant()` API.
-As an example, the following code gives `elasticfilesystem:ClientWrite` permissions to an IAM role.
+As an example, the following code gives `elasticfilesystem:Backup` permissions to an IAM role.
```

Comment:
To differentiate it from client authentication use cases, we changed the `grant()` example from `elasticfilesystem:clientWrite` to `elasticfileSystem:Backup'.